tracee/tracee.bpf.c

```diff
@@ -497,14 +500,39 @@ static __always_inline int get_config(u32 key)
     return *config;
 }
 
+// returns 1 if you should trace based on uid, 0 if not
 static __always_inline int uid_filter_matches()
 {
+
+    int equal;
+    int not_equal;
+
     uid_t uid = (u32)bpf_get_current_uid_gid();
-    u32* isMatch = bpf_map_lookup_elem(&uid_filter, &uid);
-    if (isMatch == NULL) {
-        return 0;
+    s64* equality = bpf_map_lookup_elem(&uid_equal_filter, &uid);
+    if (equality == NULL) {
+        equal = 0;
+        not_equal = 0;
+    } else {
+        not_equal = *equality == 0;
+        equal = *equality == 1;
     }
-    return 1;
+
+    u32 greater = GREATER;
+    u32 less = LESS;
+
+    s64* greaterThan = bpf_map_lookup_elem(&uid_compare_filter, &greater);
+    s64* lessThan = bpf_map_lookup_elem(&uid_compare_filter, &less);
+
+    int accumulator = !not_equal;
```

Comment:
I don't think this variable is needed as well, according to my previous and next comments